.openpublishing.redirection.json

```diff
@@ -227,143 +227,143 @@
 // ============================================================================
 // Progressive Web Apps
       {
-        "source_path": "microsoft-edge/progressive-web-apps/index.md",
+        "source_path": "./microsoft-edge/progressive-web-apps/index.md",
         "redirect_url": "/microsoft-edge/progressive-web-apps-chromium/index",
         "redirect_document_id": false
       },
       {
-        "source_path": "microsoft-edge/progressive-web-apps-chromium/serviceworker.md",
+        "source_path": "./microsoft-edge/progressive-web-apps-chromium/serviceworker.md",
         "redirect_url": "/microsoft-edge/progressive-web-apps-chromium/how-to/service-workers",
```

Comment:
Outside of the scope of this PR: I'm finding interesting cases where we have "chained" redirects. For example, the redirect rule I selected here goes from `./microsoft-edge/progressive-web-apps-chromium/serviceworker.md` to `/microsoft-edge/progressive-web-apps-chromium/how-to/service-workers`. But then, later in this file, on line 270, we have a redirect rule that goes from `./microsoft-edge/progressive-web-apps-chromium/how-to/service-workers.md` to `https://developer.mozilla.org/docs/Web/API/Service_Worker_API/Using_Service_Workers`.

So, in a separate PR, we should analyze where we have other cases like this one and eliminate the extra redirect jump (i.e. instead of going from A to B, and then from B to C, just go from A to C directly).